examples/java/direct/build.gradle

```diff
@@ -0,0 +1,28 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * License); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import groovy.json.JsonOutput
+
+apply plugin: org.apache.beam.gradle.BeamModulePlugin
+// Disable default shadow jar closure and include all class files and resources.
+applyJavaNature(shadowClosure: {})
```

Comment:
We should disable publishing as we don't want these artifacts to be part of the release.

Reply:
Having release artifacts was actually one of the motivations for this. This makes it very easy to pull a runner-enabled example binary and run it against the beam runner of your choice, which is useful for people experimenting with Beam for the first time or experimenting with different runners.

@axelmagn is working on integration testing for the Flink runner on a real deployment; having these published binaries facilitates that, though we could in theory build a custom jar for tests.

Reply:
Yeah, I'd like to challenge and ask why we wouldn't want these to be part of the release.  It would already be useful in my integration tests, and I'd imagine many beginner users would find a known-green fat jar to be useful for debugging their setups.

Reply:
This is a subset of the maven archetypes functionality that already exists since the maven archetypes allows people to edit the code and is useful for them to build their own pipelines.

Also, in my opinion Maven should only contain libraries and not applications. It is dangerous to release jars that contain code which contains unrepackaged libraries to Maven. This is an issue in several prominent projects where they include guava or some other version inside their jar leading to user confusion because of MethodNotFoundErrors. In that regard I would be fine with having these available on the Beam website as direct downloads in addition to the archetypes quickstart.